integreat_cms/cms/views/events/event_actions.py

```diff
@@ -72,6 +72,9 @@ def copy(
     region = request.region
     event = get_object_or_404(region.events, id=event_id)
 
+    if event.external_calendar:
```

Comment:
@david-venhoff Is this what you had in mind?

Reply:
Almost, but I think it would be preferable to have this check in the `event.copy` method. The advantage would be, that with the current solution, we would have to keep in mind to always check the external calendar if we wanted to call `event.copy` from a new solution in the future.

`event.copy` could return an error or bool indicating whether the copy was created successfully.

Reply:
Yes, very true. I added an Exception that is thrown when one attempts to copy an event that shouldn't be copied. Can you please check whether or not I followed the conventions and placement of the exception? :)

Reply:
Could you also document that this method can now throw an exception?

Reply:
Should be applied now :)